core/src/main/java/com/linecorp/armeria/common/DefaultCookie.java

```diff
@@ -117,6 +121,23 @@ public String sameSite() {
         return sameSite;
     }
 
+    @Override
+    public boolean isHostOnly() {
+        return hostOnly;
+    }
+
+    @Override
+    public boolean isExpired() {
+        if (maxAge == UNDEFINED_MAX_AGE) {
+            return false;
+        }
+        if (maxAge <= 0) {
+            return true;
+        }
+        final double timePassed = (System.currentTimeMillis() - createdMillis) / 1000.0;
```

Comment:
- `TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis() - createdMillis)`?
- Don't we need to consider `Expires`?

Reply:
Seems like `expires` is transformed into `maxAge` at the end of parsing. There's no `expires` in `Cookie` also.
https://github.com/line/armeria/blob/4df49cd7f8f022e505226de54663adca4080ad3b/core/src/main/java/com/linecorp/armeria/common/ClientCookieDecoder.java#L248-L252